tests.py

```diff
@@ -233,7 +234,7 @@ def test_terminate_cluster(self, id):
         except Exception as e:
             self.ERRORS += 1
             self.print_fail(name, e)
-    
+
     def test_get_jobs(self):
```

Comment:
In this `test_get_jobs` function, could you please change
`assert len(jobs) == max_response` to use `<=` instead? I know it's an irrelevant change but I just noticed this and it would make the tests work out of the box.

Reply:
Thanks for checking it out @danmun. Should I also bump the version in setup.py in this PR or is this something that will/should be done separately?

Reply:
Yes, good call. Please bump the version to `3.0.0`. Also, could you please remove `'Programming Language :: Python :: 2.7'` from the `PACKAGE_CLASSIFIERS` array in setup.py?

Reply:
Done!